### ISSUE TITLE
Question of Korea glyph design principle.

### ISSUE THREAD

**Issue by dadasutasudadas** [adobe-fonts/source-han-serif#58]
I wanted to know about the character form principle of Source Hans Serif K.
While using the fonts, I noticed that the Korean characters seem different from the Source Han Sans fonts.
For example, the right “radical” in 掅 (U+6385) is different between Sans and Serif.
I want to use these fonts to the full, so I will appreciated it very much if I could know the Glyph design principle.
![korea6385](https://cloud.githubusercontent.com/assets/21096794/25738749/cbfa38be-31b9-11e7-981e-30321dc22dee.JPG)

**Comment by kenlunde** [comment]
The answer to your question needs to be answered at several levels.

First and foremost, U+6385 掅 is outside the scope of the KS X 1001 and KS X 1002 standards, meaning that its shape is not intended to follow KR design principles. It's actually included as a JP glyph that is above and beyond those in [Adobe-Japan1-6](https://wwwimages2.adobe.com/content/dam/Adobe/en/devnet/font/pdfs/5078.Adobe-Japan1-6.pdf). _Source Han Serif_ includes JP, CN, and TW glyphs for this character, while _Source Han Sans_ includes only CN and TW ones.

So, at the character code level, and in terms of ideographs, _Source Han Sans_ and _Source Han Serif_ support GB 18030 (Chinese), Big Five (Taiwan), JIS X 0208/0212/0213 (Japan) plus additional Adobe-Japan1-6 ideographs plus additional JP ideographs, and KS X 1001/1002. Hong Kong SCS (Hong Kong) is currently supported only by _Source Han Sans_, but its support will be added for _Source Han Serif_ Version 2.000.

At the glyph level, we are faced with the 64K glyph limit, and when one needs to support all of the standards listed above, along with multiple glyphs per code point and glyphs for other characters, such as Korean hangul, this limit is easily reached.

For _Source Han Sans_, the number of additional JP ideographs that we originally intended to include was nearly 6,000, but over 4,000 of them needed to be removed prior to release, due to the 64K glyph limit. For _Source Han Serif_, we actually include almost all of the additional JP ideographs, which is why your friend U+6385 掅 appears in a form that is suitable for KR use (so-called Japanese traditional forms are almost always suitable as a Korean standard form).

Moving on to Version 2.000 for both families, I estimated that approximately 1,400 new HK glyphs will be necessary to support Hong Kong SCS in _Source Han Sans_ (approximately 1,200 existing HK glyphs will need to be modified). I identified approximately 800 ideographs that can share their glyphs, along with an additional 600 or so of the additional JP ideographs that can be removed.

Version 2.000 of _Source Han Serif_, on the other hand, will require approximately 4,500 additional HK glyphs to support Hong Kong SCS, because there really isn't any meaningful HK support in Version 1._xxx_. For Version 1.001, there will be 750 glyphs in reserve, and I marked over 4,000 of the additional JP glyphs for removal. Unfortunately, your friend U+6385 掅 is among those additional JP glyphs that are marked for removal.

In summary, the differences between the two families can be the result of errors in one of the families, or because of the necessary difference in the number of additional JP glyphs that are included, though the latter will likely become less of a difference for Version 2.000.

All of the above represent the reality with which one must deal when developing Pan-CJK typefaces whose glyphs must fit within a single font resource.

**Comment by jimmymasaru** [comment]
Could you please consider releasing these removed glyphs as standalone PostScript files in the future releases?

**Comment by frankrolf** [comment]
Any removed glyphs will all still be available in the git history.
